ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-aro-4.13-nightly
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-aro-cluster-density-v2
+  cron: 0 12 * * 3
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_WORKER_COUNT: "6"
+      ARO_WORKER_VM_SIZE: Standard_D8s_v3
```

Comment:
These are pretty beefy workers for CI

Reply:
@jtaleric I went with the same size that we are currently testing with. 
Reference: https://gitlab.cee.redhat.com/aosqe/ci-profiles/-/blob/master/scale-ci/4.13/IPI-on-Azure-OVN.install.yaml